api/Onboard.go

```diff
@@ -0,0 +1,103 @@
+package api
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/operate-first/opfcli/constants"
+	"github.com/operate-first/opfcli/models"
+	"github.com/operate-first/opfcli/utils"
+)
+
+// TODO: use kustomization package so we can unmarshal kustomizations properly, and not just resources or : https://pkg.go.dev/github.com/williepaul/kustomize/api/types
+
+func (api *API) Onboard(path string, projectDisplayName string, DisableLimitrange bool) error {
+	onboardRequest, err := models.OnboardRequestFromYAMLPath(path)
+
+	if err != nil {
+		return err
+	}
+
+	if err := api.CreateGroup(
+		onboardRequest.TeamName,
+		onboardRequest.Users,
+		true,
+	); err != nil {
+		return err
+	}
+
+	if err := api.CreateRoleBinding(onboardRequest.TeamName, "admin"); err != nil {
+		return err
+	}
+
+	var tempQuota string
+	var nsPaths []string
+	emptyCustomQuota := models.CustomResourceQuota{}
+
+	for i := range onboardRequest.Namespaces {
+		if onboardRequest.Namespaces[i].CustomQuota != emptyCustomQuota {
+			tempQuota = ""
+		} else if onboardRequest.Namespaces[i].Quota != "" && onboardRequest.Namespaces[i].Quota != "_No response_" {
```

Comment:
`_No response_` ? Why would we check for this?

Reply:
That was specific to the response I was getting from the webhook that would hook into a new `operate-first/support onboard_to_cluster` issue request. This should be moved to the Tekton side of things, removing.